### PR TITLE
Add a "SLOW MODE" banner in checked mode

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
Fixes https://github.com/flutter/flutter/issues/1682

File: packages/flutter/lib/src/widgets/checked_mode_banner.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  static final Rect kRect = new Rect.fromLTWH(-kOffset, kOffset-kHeight, kOffset * 2.0, kHeight);
+  static const TextStyle kTextStyles = const TextStyle(
+    color: const Color(0xFFFFFFFF),
+    fontSize: kFontSize,
+    fontWeight: FontWeight.w900,
+    textAlign: TextAlign.center
+  );
+
+  static final TextPainter textPainter = new TextPainter()
+    ..text = new StyledTextSpan(kTextStyles, <TextSpan>[new PlainTextSpan('SLOW MODE')])
```
Comment (eseidelGoogle) [inline packages/flutter/lib/src/widgets/checked_mode_banner.dart]: We'll eventually intl this when we intl the rest of the framework.

**Comment by eseidelGoogle** [comment]
@abarth or @HansMuller are better reviewers than me, but sgtm!

File: packages/flutter/lib/src/widgets/checked_mode_banner.dart [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  void paint(Canvas canvas, Size size) {
+    final Paint paintShadow = new Paint()
+      ..color = const Color(0x7F000000)
+      ..maskFilter = new ui.MaskFilter.blur(ui.BlurStyle.normal, kShadowBlur);
+    final Paint paintBanner = new Paint()
+      ..color = kColor;
+    canvas.translate(size.width, 0.0);
+    canvas.rotate(math.PI/4);
+    canvas.drawRect(kRect, paintShadow);
+    canvas.drawRect(kRect, paintBanner);
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/checked_mode_banner.dart]: `..`

**Comment by Hixie** [comment]
Updated to address verbal comments. PTAL.

**Comment by abarth** [comment]
LGTM

**Pull request merged**